.gitignore

```diff
@@ -14,6 +14,9 @@
 #################################
 /nbproject
 .idea
+/.project
+/.buildpath
+/.settings/
```

Comment:
I have always been of the opinion that no IDE specific ignores should be in repo's gitignore. They belong in your global gitignore.